server/controllers/payroll/multiplePayrollIndice/parameters.config.js

```diff
@@ -610,7 +611,7 @@ async function importConfig(req, res, next) {
     let checkValidEmployee = 0;
     empPayroll.forEach(emp => {
```

Comment:
I think a nicer check/error would be to indicate to the user when the error occurred.  So something like this:

```js
// returns a list of indexes into the user-submitted array for employees that are invalid.
const invalidEmployeeIndexes = arrayDataFormatted.reduce((invalidArr, index, currentIdx) => {
  // check to see if the employee is in the list.
  const isValid = empPayroll
    .some(emp => emp.employee_reference.toLowerCase() === dt.employee.toLowerCase());
 
  return isValid ? invalidArr : [...invalidArr, currentIdx];
});

// ... later ...

if (invalidEmployeeIndexes.length > 0) {
  throw new BadRequest(
    `Error: Could not find the employees on lines ${invalidEmployeeIndexes.join(',')}.`,
    `${i18n(lang)('ERRORS.ER_BAD_CSV_FILE_EMP')}`);
}
```